tools/hsmtools.c

```diff
@@ -19,6 +19,7 @@ static void show_usage(void)
 	printf("./hsmtools <method> [arguments]\n");
 	printf("methods:\n");
 	printf("	- decrypthsm <path/to/hsm_secret> <password>\n");
+	printf("	- encrypthsm <path/to/hsm_secret> <password>\n");
```

Comment:
Since the tool is call htmtools, we could just call these commands "decrypt" and "encrypt" perhaps?  And maybe "hsmtool" is better than "hsmtools" though that's marginal.

Reply:
Yeah I wanted to add more tool*s* (https://github.com/ElementsProject/lightning/pull/3217) but I can make it singular ^^